src/integrationTest/java/org/opensearch/test/framework/cluster/SearchRequestFactory.java

```diff
@@ -26,14 +26,21 @@ public final class SearchRequestFactory {
 	private SearchRequestFactory() {
 
 	}
-	public static SearchRequest queryStringQueryRequest(String indexName, String queryString) {
+	public static SearchRequest queryStringQueryRequest(String indexName, String queryString, Boolean ccsMinimizeRoundtrips) {
```

Comment:
The SearchRequest is mutable, these additional overloads seem like they are going to be more maintenance work as additional scenarios are added over the long run.  What would you think about creating a 'builder' function like `minimizeRoundTrips(...)` that can be used inline to add these parameter?

```java
public static SearchRequest minimizeRoundTrips(SearchRequest sr) {
   sr.setCcsMinimizeRoundtrips(true);
   return sr;
}
```

Reply:
done